cmake/thirdparty/SetupOpenMP.cmake

```diff
@@ -40,13 +40,24 @@ endif()
 
 # Allow user to override
 if (BLT_OPENMP_COMPILE_FLAGS)
```

Comment:
Note: This `if()` conditional should only be active when there is a `BLT_OPENMP_COMPILE_FLAGS` variable in scope, which should only happen if it is defined in your host-config (or the application build system that brings in blt).

Reply:
both BLT_OPENMP_COMPILE_FLAGS and BLT_OPENMP_LINK_FLAGS get autofilled. Nothing I added manually.